tests/test_pipelines.py

```diff
@@ -492,6 +492,12 @@ def test_stable_diffusion_pndm(self):
         assert np.abs(image_slice.flatten() - expected_slice).max() < 1e-2
         assert np.abs(image_from_tuple_slice.flatten() - expected_slice).max() < 1e-2
 
+    def test_from_pretrained_error_message_uninstalled_packages(self):
```

Comment:
@pcuenca - I can't think of a good way to test that we have a nice error message going forward. Suggest to just merge without a good test for now. Let me know if you have other ideas!

Reply:
The test is not as important as the real use in the wild, like what happened with that Space that broke. If users see something reasonable instead of the previous cryptic message, then I'm all for it!

Reply:
Yeah, I think the test as it's now doesn't really help at all :sweat_smile: I was just wondering if there is a way to "pretend" `scipy` is not installed and then verify that the error message is nice